internal/controller/nginx/config/base_http_config.go

```diff
@@ -38,3 +46,22 @@ func executeBaseHTTPConfig(conf dataplane.Configuration) []executeResult {
 
 	return results
 }
+
+func buildAccessLog(accessLogConfig *dataplane.AccessLog) *AccessLog {
+	if accessLogConfig != nil {
+		accessLog := &AccessLog{
+			Path:       dataplane.DefaultAccessLogPath,
+			FormatName: dataplane.DefaultLogFormatName,
+		}
+		if accessLogConfig.Format != "" {
+			accessLog.Format = accessLogConfig.Format
+		}
+
+		if accessLogConfig.Disabled {
+			accessLog.Disabled = accessLogConfig.Disabled
+		}
```

Comment:
nit: can simplify this to
```suggestion
		accessLog.Disabled = accessLogConfig.Disabled
```

Reply:
i would set it only if Disabled = true, i know that it was cleaned in configuration and also weill be cleaned up in template, but i'm always try to add more checks :D